content/intro-to-storybook/vue/zh-CN/deploy.md

```diff
@@ -0,0 +1,149 @@
+---
+title: '部署Storybook'
+tocTitle: '部署'
+description: '学习如何部署Storybook'
+commit: '107b7ce'
+---
+
+在此教程中，我们在自己的电脑上构建了组件。某些时候我们还需要和组员分享我们的工作以获得其他人的反馈。让我们部署 Storybook 让其他成员审查实现好的 UI 吧。
+
+## 导出为一个静态应用
+
+为了部署 Storybook 我们首先需要将其导出为一个静态 web 应用。Storybook 已经集成了此功能并已预配置完成，我们只需要将[开始吧章节](/intro-to-storybook/vue/zh-CN/get-started)中的脚本更新即可。
+
+```json
+{
+  "scripts": {
+    "build-storybook": "build-storybook -s public"
+  }
+}
+```
```

Comment:
@zydmayday could you do another pass at this section? It seems that this section or some portions of it are a bit outdated! Compare it with the English version and adjust accordingly if you don't mind.